x-pack/plugins/security_solution/public/detections/components/alerts_kpis/alerts_count_panel/index.tsx

```diff
@@ -68,13 +60,10 @@ export const AlertsCountPanel = memo<AlertsCountPanelProps>(
     filters,
     inspectTitle,
     panelHeight,
-    query,
-    runtimeMappings,
     setStackByField0,
     setStackByField0ComboboxInputRef,
     setStackByField1,
     setStackByField1ComboboxInputRef,
-    signalIndexName,
```

Comment:
`runtimeMappings` and `signalIndexName` are not used anymore, can we clean them from the component prop type?